### PR TITLE
Add information on ``skip_slack_parsing``

### PR DESCRIPTION
**Pull request by somenet**
Accompanying doc change to https://github.com/mattermost/mattermost-server/pull/13320

### CODE REVIEW

File: site/content/integrate/slash-commands/_index.md [hunk 1]
```diff
@@ -88,6 +88,8 @@ Slash command responses support more than just the `text` field. Here is a full
 | type | Sets the post `type`, mainly for use by plugins.<br> If not blank, must begin with `custom_`. Passing `attachments` will ignore this field and set the type to `slack_attachment`. | No |
 | extra\_responses | An array of responses used to send more than one post in your response. Each item in this array takes the shape of its own command response, so it can include any of the other parameters listed here, except `goto\_location` and `extra\_responses` itself. Available in Mattermost v5.6 and later. | No |
 | props | Sets the post `props`, a JSON property bag for storing extra or meta data on the post.<br> Mainly used by other integrations accessing posts through the REST API.<br> The following keys are reserved: `from_webhook`, `override_username`, `override_icon_url` and `attachments`. | No |
+| skip_slack_parsing | Signals mattermost that the post is correctly escaped for mattermost and the Slack-compatibility logic should be skipped. Useful if the post contains code which is incorrectly mangled by the Slack-compatibility logic. | No |
```
Comment (justinegeffen) [inline site/content/integrate/slash-commands/_index.md]:
```suggestion
| skip_slack_parsing | Signals Mattermost that the post is correctly escaped for Mattermost and the Slack-compatibility logic should be skipped. Useful if the post contains code which is incorrectly handled by the Slack-compatibility logic. | No |
```

I like "mangled" but in this context I think "incorrectly handled" is more accurate? If "mangled" must be used then remove "incorrectly" and just have "mangled": "Useful if the post contains code which is mangled by the Slack-compatibility logic."

File: site/content/integrate/slash-commands/_index.md [hunk 2]
```diff
@@ -88,6 +88,8 @@ Slash command responses support more than just the `text` field. Here is a full
 | type | Sets the post `type`, mainly for use by plugins.<br> If not blank, must begin with `custom_`. Passing `attachments` will ignore this field and set the type to `slack_attachment`. | No |
 | extra\_responses | An array of responses used to send more than one post in your response. Each item in this array takes the shape of its own command response, so it can include any of the other parameters listed here, except `goto\_location` and `extra\_responses` itself. Available in Mattermost v5.6 and later. | No |
 | props | Sets the post `props`, a JSON property bag for storing extra or meta data on the post.<br> Mainly used by other integrations accessing posts through the REST API.<br> The following keys are reserved: `from_webhook`, `override_username`, `override_icon_url` and `attachments`. | No |
+| skip_slack_parsing | Signals Mattermost that the post is correctly escaped for Mattermost and the Slack-compatibility logic should be skipped. Useful if the post contains code which is incorrectly handled by the Slack-compatibility logic. | No |
```
Comment (hanzei) [inline site/content/integrate/slash-commands/_index.md]: Would you please move this in front of `props`?

Comment (hanzei) [inline site/content/integrate/slash-commands/_index.md]: The documentation doesn't make it clear what the valid values are. Would you please make this clear?

Comment (somenet) [reply site/content/integrate/slash-commands/_index.md]: ACK. For future reference: what is the a reason for it? How are the entries ordered?

Comment (hanzei) [reply site/content/integrate/slash-commands/_index.md]: `Props` is like the "appendix". All information that doesn't belong in a specific field gets added there. This is why it should be last.

**Comment by jasonblais** [comment]
Adding a `Lifecycle/Frozen` label while the server PR is being reviewed https://github.com/mattermost/mattermost-server/pull/13420

**Pull request merged**